src/server/transaction.cc

```diff
@@ -256,7 +256,7 @@ void Transaction::InitByKeys(KeyIndex key_index) {
     shard_data_.front().local_mask |= ACTIVE;
 
     unique_shard_cnt_ = 1;
-    unique_shard_id_ = Shard(args_.front(), shard_set->size());
+    unique_shard_id_ = Shard(args_.front(), shard_set->size());  // TODO: Squashed bug
```

Comment:
what's this todo?

Reply:
There are commands where the first argument is not a key, but the number of keys or smth else. For example, ZINTERCARD. But we don't support it yet